# try-catch #

- 一般try-catch是捕获服务端的报错,但是近年来客户端也应该重视起来,finall最终执行连return也无法阻止

        try {
            someFunction();
        } catch (error) {
            if (error instanceof TypeError) {
                //处理类型错误
            } else if (error instanceof ReferenceError) {
                //处理引用错误
            } else {
                //处理其他类型的错误
            }
        }
